packages/block-editor/src/components/url-input/index.js

```diff
@@ -223,9 +223,9 @@ class URLInput extends Component {
 		this.inputRef.current.focus();
 	}
 
-	static getDerivedStateFromProps( { showSuggestionsOverride }, { showSuggestions } ) {
```

Comment:
Aren't we breaking back compatibility with current usages of showSuggestionsOverride? Should we keep supporting the existing prop but write a deprecated warning when it is used?

Reply:
It was added in #10128 (merged 16 hours ago) so I think we don't break anything.